hack/utils

```diff
@@ -77,10 +77,12 @@ install_nginx_ingress() {
     sleep 1
   done
```

Comment:
exactly why this happens given the above check that waits for pods ^?  I don't know :)

Reply:
https://github.com/jumpstarter-dev/jumpstarter-controller/pull/186

Alternative, let's see how it works.